201-servicebus-create-eventhub-and-consumergroup/azuredeploy.json

```diff
@@ -5,7 +5,7 @@
         "serviceBusNamespaceName": {
```

Comment:
The provider is still Microsoft.ServiceBus. Messaging and EventHub type namespace fall under the umbrella of ServiceBus in general.